proms-resources/feature/src/main/features/feature.json

```
@@ -70,7 +70,9 @@
         "io.uhndata.cards.proms-backend:VisitFormsPreparation=[patient-visit-backend]",
         "io.uhndata.cards.proms-permissions:UnsubmittedFormsRestriction=[sling-readall]",
         "io.uhndata.cards.proms-backend:TorchImporter=[proms-import-backend]",
-        "io.uhndata.cards.proms-backend:MetricLogger=[cards-metrics]"
+        "io.uhndata.cards.proms-backend:MetricLogger=[cards-metrics]",
+        "io.uhndata.cards.proms-backend:SlackNotifications=[cards-metrics]",
+        "io.uhndata.cards.proms-backend=[sling-readall]"
```

After following my other comment about using a sub-service, the following change would be required:
```suggestion
        "io.uhndata.cards.proms-backend:CsvExport=[sling-readall]"
```
Additionally, instead of using `sling-readall` you might want to create a `csv-export` _service user_ that is restricted to only reading the JCR nodes that it needs to read.